vendor/foundation_rails_helper/form_builder.rb

```diff
@@ -0,0 +1,245 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2015 Sébastien Gruhier (http://xilinus.com/)
+# MIT License
```

Comment:
While these lines are OK, the license says:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

So what do you think about copying the LICENSE file from the foundation_rails_helper project to `vendor/foundation_rails_helper/`?